homeassistant/components/schlage/entity.py

```diff
@@ -0,0 +1,42 @@
+"""Base entity class for Schlage."""
+
+from pyschlage.lock import Lock
+
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN, MANUFACTURER
+from .coordinator import SchlageDataUpdateCoordinator
+
+
+class SchlageEntity(CoordinatorEntity[SchlageDataUpdateCoordinator]):
+    """Base Schlage entity."""
+
+    _attr_has_entity_name = True
+    _attr_name = None
```

Comment:
I think this should only be on the lock so the sensor can get it's name from the device class